ckan/lib/create_test_data.py

```diff
@@ -62,18 +76,17 @@ def create_test_user(cls):
 
     @classmethod
     def create_translations_test_data(cls):
-        import ckan.model
-        CreateTestData.create()
+        _create_test_data()
 
-        sysadmin_user = ckan.model.User.get('testsysadmin')
-        package = ckan.model.Package.get('annakarenina')
+        sysadmin_user = model.User.get('testsysadmin')
+        package = model.Package.get('annakarenina')
```

Comment:
Perhaps rather than catch an SQLAlchemy error and rollback the session it's better to check if the data has been created before attempting to create it again

```suggestion
        package = model.Package.get('annakarenina')
        if not package:
             CreateTestData.create()
```

Reply:
@amercader i thought the same, but then, we will have redundant code:

```python
sysadmin_user = model.User.get('testsysadmin')
package = model.Package.get('annakarenina')
if not package:
   CreateTestData.create()
   sysadmin_user = model.User.get('testsysadmin')
   package = model.Package.get('annakarenina')
```
if you agree with this i will make the changes